lib/collapsers/binary.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var logger = require('bole')('collapsify:collapsers:binary');
+var base64Utils = require('../utils/base-64');
+
+function external(url, opts) {
+  logger.info('Fetching external binary from "%s".', url);
+
+  return opts.fetch(url).then(collapse);
```

Comment:
`opts.fetch` is a the result of creating the HTTP client. I could make it a global or a singleton, but that makes it annoying to pass in a mock function in tests.